src/node/routes/index.ts

```diff
@@ -121,23 +121,29 @@ export const register = async (
     throw new HttpError("Not Found", HttpCode.NotFound)
   })
 
-  const errorHandler: express.ErrorRequestHandler = async (err, req, res, next) => {
-    const resourcePath = path.resolve(rootPath, "src/browser/pages/error.html")
-    res.set("Content-Type", getMediaMime(resourcePath))
-    try {
+  const errorHandler: express.ErrorRequestHandler = async (err, req, res) => {
+    if (err.code === "ENOENT" || err.code === "EISDIR") {
+      err.status = HttpCode.NotFound
+    }
+
+    const status = err.status ?? err.statusCode ?? 500
```

Comment:
Any reason we don't just use `||`? Is it best practice now to use `??` to avoid `0` being treated as false?

Reply:
I'm not sure if it's best practice, but the `0` is why I used it. Although that's probably not a valid HTTP code anyway?

Reply:
Yea it isnt.

Reply:
nbd, just asking questions.